src/content/it/fundamentals/performance/http2/index.md

```diff
@@ -0,0 +1,681 @@
+nuove opportunità per ottimizzare le nostre applicazioni e migliorare 
+le prestazioni!
+
+Gli obiettivi primari per HTTP/2 sono ridurre la latenza attivando il 
+multiplexing di richiesta e risposta, minimizzare il sovraccarico di 
+protocollo tramite la compressione efficiente dei campi di 
+intestazione HTTP e aggiungere il supporto per la priorità delle 
+richieste e il server  push.
+Per implementare questi requisiti, esiste un grande supporto di altri 
+miglioramenti del protocollo, come il nuovo controllo del flusso, il 
+trattamento degli errori e i meccanismi di aggiornamento, ma queste 
+sono le caratteristiche più importanti che ogni sviluppatore web deve 
+comprendere e sfruttare nelle proprie applicazioni.
+
+HTTP/2 non modifica in alcun modo la semantica dell'applicazione di 
+HTTP. Tutti i concetti fondamentali, come metodi HTTP, codici di 
+stato, URI e campi di intestazione, rimangono in vigore. Invece, 
+HTTP/2 modifica come i dati vengono formattati (framed) e trasportati 
+tra il client e il server, che gestiscono l'intero processo e 
+nascondono tutta la complessità delle nostre applicazioni all'interno 
+del nuovo livello di framing. Di conseguenza, tutte le applicazioni 
+esistenti possono essere consegnate senza modifiche.
+
+*Perché non HTTP/1.2?*
+
+Per raggiungere gli obiettivi di performance impostati dal gruppo di 
+lavoro HTTP, HTTP/2 introduce un nuovo livello di framing binario che 
+non è compatibile con i server e i client precedenti HTTP/1.x, da qui 
+l'incremento maggiore della versione di protocollo in HTTP/2.
+
+Detto questo, a meno che non si stia implementando un server web (o un 
+client personalizzato) lavorando con socket TCP raw, non si vedrà 
+alcuna differenza: tutte le novità e il framing di basso livello viene 
+eseguito dal client e dal server per tuo conto. Le uniche differenze 
+osservabili saranno un miglioramento delle prestazioni e la 
+disponibilità di nuove funzionalità come la priorità nelle richieste, 
+il controllo del flusso e il server push.
+
+## Una breve storia di SPDY e HTTP/2
+
+SPDY è stato un protocollo sperimentale, sviluppato da Google e 
+annunciato a metà del 2009, il cui obiettivo primario era quello di 
+cercare di ridurre la latenza di caricamento delle pagine web, 
+affrontando alcune delle famose limitazioni di prestazioni di 
+HTTP/1.1.
+In particolare, gli obiettivi del progetto sono 
+stati i seguenti:
+
+* Obiettivo una riduzione del 50% del tempo di caricamento della 
+pagina (PLT).
+* Evitare la necessità di modifiche al contenuto dagli autori del sito 
+web.
+* Ridurre al minimo la complessità di distribuzione e evitare le 
+modifiche delle infrastrutture di rete.
+* Sviluppare questo nuovo protocollo in collaborazione con la comunità 
+open-source.
+* Raccogliere dati reali di prestazioni per convalidare o meno il 
+protocollo sperimentale.
+
+Note: Per ottenere il miglioramento del PLT del 50%, SPDY mirava a 
```

Comment:
You doesn't translate any "Note" to "Nota" because it is a special [markdown callouts](https://developers.google.com/web/resources/markdown-syntax#callouts).